clang/include/clang/Lex/Preprocessor.h

```diff
@@ -2408,6 +2428,14 @@ class Preprocessor {
              bool *IsFrameworkFound, bool SkipCache = false,
              bool OpenFile = true, bool CacheFailures = true);
 
+  /// Given a "foo" or \<foo> reference, look up the indicated embed resource.
+  ///
+  /// Returns std::nullopt on failure.  \p isAngled indicates whether the file
+  /// reference is for system \#include's or not (i.e. using <> instead of "").
```

Comment:
Can you document the OpenFile and LookupFromFile arguments as well?

Reply:
Sure can! That's done now.